### ISSUE TITLE
win32-1.7.4+1/lib/src/structs.dart:1111:7: Error: Struct 'ENUMLOGFONTEX' is empty. Empty

### ISSUE THREAD

**Issue by xiaocode337317439** [flutter/flutter#76777]
Flutter 1.27.0-9.0.pre.85 • channel master • git@github.com:flutter/flutter.git
Framework • revision 71b2b4d6e0 (3 hours ago) • 2021-02-25 00:01:01 -0500
Engine • revision 867e51b762
Tools • Dart 2.13.0 (build 2.13.0-73.0.dev)

Launching lib/main.dart on iPhone 12 Pro Max in debug mode...
Running pod install...
Running Xcode build...
Xcode build done.                                           22.2s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **

Xcode's output:
↳
    ../../../../.pub-cache/hosted/pub.flutter-io.cn/win32-1.7.4+1/lib/src/structs.dart:1111:7: Error: Struct 'ENUMLOGFONTEX' is empty. Empty structs are undefined behavior.
    class ENUMLOGFONTEX extends Struct {
          ^
    ../../../../.pub-cache/hosted/pub.flutter-io.cn/win32-1.7.4+1/lib/src/structs.dart:2835:7: Error: Struct 'BLUETOOTH_PIN_INFO' is empty. Empty structs are undefined behavior.
    class BLUETOOTH_PIN_INFO extends Struct {
          ^
    ../../../../.pub-cache/hosted/pub.flutter-io.cn/win32-1.7.4+1/lib/src/structs.dart:2960:7: Error: Struct 'EXCEPINFO' is empty. Empty structs are undefined behavior.
    class EXCEPINFO extends Struct {}
          ^
    ../../../../.pub-cache/hosted/pub.flutter-io.cn/win32-1.7.4+1/lib/src/structs.dart:2966:7: Error: Struct 'PROPERTYKEY' is empty. Empty structs are undefined behavior.
    class PROPERTYKEY extends Struct {}
          ^
    ../../../../.pub-cache/hosted/pub.flutter-io.cn/win32-1.7.4+1/lib/src/structs.dart:2973:7: Error: Struct 'PROPVARIANT' is empty. Empty structs are undefined behavior.
    class PROPVARIANT extends Struct {}
          ^
    ../../../../.pub-cache/hosted/pub.flutter-io.cn/win32-1.7.4+1/lib/src/structs.dart:2978:7: Error: Struct 'SAFEARRAY' is empty. Empty structs are undefined behavior.
    class SAFEARRAY extends Struct {}
          ^
    ../../../../.pub-cache/hosted/pub.flutter-io.cn/win32-1.7.4+1/lib/src/structs.dart:2985:7: Error: Struct 'CLSID' is empty. Empty structs are undefined behavior.
    class CLSID extends Struct {}
          ^
    ../../../../.pub-cache/hosted/pub.flutter-io.cn/win32-1.7.4+1/lib/src/structs.dart:2992:7: Error: Struct 'STATSTG' is empty. Empty structs are undefined behavior.
    class STATSTG extends Struct {}
          ^
    ../../../../.pub-cache/hosted/pub.flutter-io.cn/win32-1.7.4+1/lib/src/structs.dart:2999:7: Error: Struct 'NLM_SIMULATED_PROFILE_INFO' is empty. Empty structs are undefined behavior.
    class NLM_SIMULATED_PROFILE_INFO extends Struct {}
          ^
    ../../../../.pub-cache/hosted/pub.flutter-io.cn/ffi-0.1.3/lib/src/utf8.dart:23:7: Error: Struct 'Utf8' is empty. Empty structs are undefined behavior.
    class Utf8 extends Struct {
          ^
    ../../../../.pub-cache/hosted/pub.flutter-io.cn/ffi-0.1.3/lib/src/utf16.dart:16:7: Error: Struct 'Utf16' is empty. Empty structs are undefined behavior.
    class Utf16 extends Struct {
          ^
    ../../../../.pub-cache/hosted/pub.flutter-io.cn/ffi-0.1.3/lib/src/allocation.dart:47:33: Error: Expected type 'T' to be a valid and instantiated subtype of 'NativeType'.
      final int totalSize = count * sizeOf<T>();
                                    ^

    Command PhaseScriptExecution failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: no rule to process file '/Users/m6/.pub-cache/git/native_plugin-51f134f83c7966668570312d04798db4382701d5/ios/Classes/libCPU.a' of type 'archive.ar' for architecture 'x86_64' (in target 'native_plugin' from project 'Pods')

Could not build the application for the simulator.
Error launching application on iPhone 12 Pro Max.

**Comment by dcharkes** [comment]
Duplicate of https://github.com/flutter/flutter/issues/76705, the dependencies need to be rolled forward.

**Comment by darshankawar** [comment]
Closing this as duplicate of https://github.com/flutter/flutter/issues/76705
